exercises/exercise-2/exercise-2.js

```diff
@@ -70,3 +70,29 @@ let hogwarts = [
     occupation: "Teacher",
   },
 ];
+
+// TASK 1
+
+for (let {
+  firstName: name1,
+  lastName: name2,
+  house: schoolHouse,
+} of hogwarts) {
+  if (schoolHouse === "Gryffindor") {
+    console.log(`${name1} ${name2}`);
+  }
+}
```

Comment:
This is good if you want to use different variable names to the ones defined in the object, but you could just do this too:

```suggestion
for (let {
  firstName,
  lastName,
  house,
} of hogwarts) {
  if (house === "Gryffindor") {
    console.log(`${firstName} ${lastName}`);
  }
}
```